conda-envs/environment-dev-py37.yml

```diff
@@ -1,29 +1,29 @@
 name: pymc3-dev-py37
 channels:
-  - conda-forge
-  - defaults
+- conda-forge
+- defaults
 dependencies:
-  - python=3.7
-  - arviz>=0.9
-  - theano-pymc==1.0.12
-  - numpy>=1.13
-  - scipy>=0.18
-  - pandas>=0.18
-  - patsy>=0.5
-  - fastprogress>=0.2
-  - h5py>=2.7
-  - typing-extensions>=3.7
-  - python-graphviz
-  - ipython>=7.16
-  - nbsphinx>=0.4
-  - numpydoc>=0.9
-  - pre-commit>=2.8.0
-  - pytest-cov>=2.5
-  - pytest>=3.0
-  - recommonmark>=0.4
-  - sphinx-autobuild>=0.7
-  - sphinx>=1.5
-  - watermark
-  - mkl-service
-  - dill
-  - libblas=*=*mkl
+- arviz>=0.9
+- dill
+- fastprogress>=0.2
+- h5py>=2.7
+- ipython>=7.16
+- libblas=*=*mkl
+- mkl-service
+- nbsphinx>=0.4
+- numpy>=1.13
+- numpydoc>=0.9
+- pandas>=0.18
+- patsy>=0.5
+- pre-commit>=2.8.0
+- pytest-cov>=2.5
+- pytest>=3.0
+- python-graphviz
+- python=3.7
+- recommonmark>=0.4
+- scipy>=0.18
+- sphinx-autobuild>=0.7
+- sphinx>=1.5
+- theano-pymc=1.0.12
```

Comment:
I thought the point is that we can drop this?

Reply:
I don't see how to drop it, I don't think it can be substituted from somewhere else. Suggestions?

So, in lieu of that, we can have a little script which checks they're all pinned to the same version

Reply:
Why not just defer to what's in requirements.txt?

Reply:
As in, leave it out of here and then do `pip install -e .`?

Sure, could do, but my understanding was that the recommended way of installing theano was via conda

Reply:
Just checked this again and [the docs](https://theano-pymc.readthedocs.io/en/latest/install_ubuntu.html) read

> We only support the installation of the requirements through conda.

So, as long as we install `mlk-service` via `conda`, then we're OK to install `theano-pymc` via pip? If so I'll do that, it's easier than adding extra scripts/checks